tests/unit_tests/pyanaconda_tests/modules/payloads/payload/test_module_payload_dnf_installation.py

```diff
@@ -474,15 +477,17 @@ def test_error_update(self, execute):
             assert any(map(lambda x: msg in x, cm.output))
 
     @patch("pyanaconda.core.util.execWithRedirect")
-    def test_multilib_policy(self, execute):
-        """Update the multilib policy."""
+    def test_multilib_policy_dnf4(self, execute):
+        """Update the multilib policy on pre-dnf5 systems."""
         execute.return_value = 0
 
         with tempfile.TemporaryDirectory() as sysroot:
             data = PackagesConfigurationData()
             data.multilib_policy = MULTILIB_POLICY_ALL
+            dnf_manager = Mock(spec=DNFManager)
+            dnf_manager.is_package_available.return_value = False
 
-            task = UpdateDNFConfigurationTask(sysroot, data)
+            task = UpdateDNFConfigurationTask(sysroot, data, dnf_manager)
             task.run()
```

Comment:
Would be great to check also if the required plugin package is correctly requested.